src/System.Net.Primitives/src/System/Net/IPEndPoint.cs

```diff
@@ -119,6 +119,157 @@ public int Port
             }
         }
 
+        public static bool TryParse(string endPointString, out IPEndPoint result)
+        {
+            if (endPointString == null)  // Avoid null ref exception on endPointString.AsSpan()
```

Comment:
It's fine to check for null, but there should not be a null ref exception here... calling AsSpan on a null string should just return a default span. Are you seeing otherwise?

Reply:
Same as above. Will fix